app/forms/teacher_interface/sanction_confirmation_form.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module TeacherInterface
+  class SanctionConfirmationForm < BaseForm
+    attribute :application_form
+    attribute :confirmed_no_sanctions
+
+    validate :no_sanctions_confirmed
+
+    private
+
+    def no_sanctions_confirmed
+      unless confirmed_no_sanctions
+        errors.add(:confirmed_no_sanctions, :not_confirmed)
+      end
+    end
+
+    def update_model
+      application_form.update(confirmed_no_sanctions:)
```

Comment:
```suggestion
      application_form.update!(confirmed_no_sanctions:)
```

I've been using `update!` since it should always pass if the form is valid.